### PR TITLE
remove UserWarning(remove learning rate from compile())

### PR DESCRIPTION
**Pull request by skyer9**
this patch removes user warnning below.
```
C:\Users\skyer9\AppData\Local\Programs\Python\Python35\lib\site-packages\keras\b
ackend\tensorflow_backend.py:2124: UserWarning: Expected no kwargs, you passed 1

kwargs passed to function are ignored with Tensorflow backend
  warnings.warn('\n'.join(msg))
Epoch 1/2
6/6 [==============================] - 0s - loss: 1.3499
Epoch 2/2
6/6 [==============================] - 0s - loss: 1.3280
```

### CODE REVIEW

**Comment by hunkim** [comment]
@kkweon What do you think?

**Comment by kkweon** [comment]
@hunkim Is it necessary to define a learning rate manually?
### If not we can just drop the learning rate keyword. 
```python
model.compile(loss='mse', optimizer='rmsprop')
```

I prefer not to define a learning rate manually because 
- it looks cleaner
- a default value is already pretty good for most of times
- unless I have a very specific reason like the default value does not work

### If necessary, then this is the only way to go. So, okay to merge

**Comment by skyer9** [comment]
always!
learning rate is important.

so, i think it is better explaining how to set learning rate
